src/components/input/bl-input.css

```diff
@@ -84,6 +89,16 @@ input {
   background-color: transparent;
 }
 
+input:disabled {
+  cursor: not-allowed;
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus {
```

Comment:
Do we really need to set hover and focus states as well?

Reply:
Not, apparently.